concordia/settings.py

```diff
@@ -0,0 +1,265 @@
+# TODO: use correct copyright header
+import os
+import sys
+
+from machina import get_apps as get_machina_apps
+from machina import MACHINA_MAIN_TEMPLATE_DIR
+from machina import MACHINA_MAIN_STATIC_DIR
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
+BASE_DIR = os.path.dirname(PROJECT_DIR)
+
+sys.path.append(BASE_DIR)
+
+sys.path.append(os.path.join(BASE_DIR, 'config'))
+
+from config import Config
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = Config.Get('django_secret_key')
+
+# Optional SMTP authentication information for EMAIL_HOST.
+EMAIL_HOST_USER = ''
+EMAIL_HOST_PASSWORD = ''
+EMAIL_USE_TLS = False
+DEFAULT_FROM_EMAIL="no-reply@loc.gov"
+
+ALLOWED_HOSTS = ['*'] # TODO: place this value in config.json
+
+if Config.mode == "production":
+    # TODO: production we can not have DEBUG = True
+    DEBUG = False
+    # TODO: For final deployment to production, when we are running https, uncomment this next line
+    #    CSRF_COOKIE_SECURE = True
+else:
+    DEBUG = True
+    CSRF_COOKIE_SECURE = False
+
+sys.path.append(PROJECT_DIR)
+AUTH_PASSWORD_VALIDATORS = []
+EMAIL_BACKEND = 'django.core.mail.backends.filebased.EmailBackend'
+# EMAIL_FILE_PATH = os.path.join(BASE_DIR, 'emails')
+EMAIL_HOST = 'localhost'
+EMAIL_PORT = 25
+LANGUAGE_CODE = 'en-us'
+LOGIN_REDIRECT_URL = '/'
+LOGOUT_REDIRECT_URL = '/'
+ROOT_URLCONF = 'concordia.urls'
+STATIC_ROOT = 'static'
+STATIC_URL = '/static/'
+STATICFILES_DIRS = [os.path.join(PROJECT_DIR, 'static'),
+                    os.path.join('/'.join(PROJECT_DIR.split('/')[:-1]), 'concordia/static')]
```

Comment:
This could be more cleanly written using `os.path.join` & `os.path.normpath`:

`os.path.normpath(os.path.join(PROJECT_DIR, "..", "concordia", "static"))`